packages/panels/docs/04-pages.md

```diff
@@ -54,6 +54,35 @@ protected function getHeaderActions(): array
 }
 ```
 
+### Opening an action modal when a page loads
+
+You can also open an action when a page loads by setting the `$defaultAction` property to the name of the action you want to open:
+
+```php
+use Filament\Actions\Action;
+
+public $defaultAction = 'onboarding';
+
+public function onboardingAction(): Action
+{
+    return Action::make('onboarding')
+        ->modalHeading('Welcome')
+        ->visible(fn (): bool => ! auth()->user()->isOnBoarded());
+}
+```
+
+You can also pass an array of arguments to the default action using the `$defaultActionArguments` property:
+
+```php
+public $defaultActionArguments = ['step' => 2];
+```
+
+Alternatively, you can open an action modal when a page loads by specifying the `defaultAction` as a query string parameter to the page:
+
+```
+/admin/products/edit/932510?defaultAction=onboarding
```

Comment:
```suggestion
/admin/products/edit/932510?action=onboarding
```

Hey @archilex, @danharrin i think this is wrong? The correct query param is `action`. `defaultAction` doesn't work as Povilas just pointed in his last video.

Reply:
I know for sure that `action` works because i'm using it in production. Not sure about `defaultAction`.

Reply:
https://github.com/filamentphp/filament/blob/e90998fe6aa4681569f09b6ebb01e5346d3a1644/packages/actions/src/Concerns/InteractsWithActions.php#L42-L49

Reply:
it was already fixed in the docs, I just haven't published them since